config/README.md

```diff
@@ -50,6 +51,17 @@ func New(path string) (Config, error)
 ```
 New takes in a path to a configuration yaml and returns a Configuration.
 
+#### type Health
+
+```go
+type Health struct {
```

Comment:
I originally had that, but the YAML marshaller didn't recognize 'health-check' in the YAML. 

I can tag it to recognize it though.  Should I do that?  
Something like:
```
HealthCheck HealthCheck `yaml:"health-check"`
```
In the Config struct.